.pylintrc

```diff
@@ -30,6 +30,7 @@ disable=
   suppressed-message,
   ungrouped-imports,
   unsubscriptable-object,
+  useless-object-inheritance, # Remove once all bots are on Python 3.
```

Comment:
There's quite a few extra lint checks in Python 3. Maybe we should just remove the lint step from Travis until then. We're already doing that for the Python 3 Cloud build CI.

Reply:
If they get to be a problem it might make sense. The other check that I disabled in one spot in fuzz task actually looked good to me, though. I just didn't want to make a more invasive change as part of this fix. I wouldn't object to disabling them if you think it makes sense, but I also don't mind keeping them around so we can slowly clean things up as we transition.